packages/token-bridge-sdk/src/hooks/arbTokenBridge.types.ts

```diff
@@ -166,10 +180,23 @@ export interface ArbTokenBridgeEth {
   }) => Promise<void | ContractReceipt>
 }
 
+export interface ArbTokenBridgeCache {
+  erc20: string[]
+  erc721: string[]
+  expire: () => void
+}
+
+export interface SearchableToken extends ERC20BridgeToken {
+  tokenLists: number[]
+}
+
 export interface ArbTokenBridgeToken {
+  tokensFromLists: ContractStorage<ERC20BridgeToken>
```

Comment:
I'd remove this from `arbTokenBridge` and move it to its own hooks. Do we need 2 separate hooks? I feel like we always access both at the same time